README.md

```diff
@@ -19,6 +19,14 @@ Events include network level information such as source & destination IPs and po
 
 ![design diagram](./agent_design.png)
 
+The agent generates events per Kubernetes host by inspecting network traffic, in the perspective of either the receiving or sending process.
+This results in events being generated in the following 3 scenarios:
+- External to pod (cluster ingress)
+- Pod to pod
```

Comment:
```suggestion
- Pod to pod
- Pod to Service
```

I think we see Pod to Service comms, too, right? If so, we'll need to also update the scenario count in the paragraph above. (Or say "… _amongst_ our scenarios are such diverse elements as:")